woocommerce.php

```diff
@@ -342,6 +342,7 @@ public function frontend_includes() {
 		include_once( WC_ABSPATH . 'includes/class-wc-shortcodes.php' );                     // Shortcodes class
 		include_once( WC_ABSPATH . 'includes/class-wc-embed.php' );                          // Embeds
 		include_once( WC_ABSPATH . 'includes/class-wc-structured-data.php' );                // Structured Data class
+		include_once( WC_ABSPATH . 'includes/class-wc-twenty-seventeen.php' );               // Twenty Seventeen support
```

Comment:
Rather than load this class and check in there, it would be better to have a single utility function or class which compares the name of the theme and includes a file if needed. Then that can be used for all themes which have a file.